receiver/filelogreceiver/filelog.go

```diff
@@ -0,0 +1,74 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package filelogreceiver
+
+import (
+
+	// Register input operator for filelog
+	_ "github.com/open-telemetry/opentelemetry-log-collection/operator/builtin/input/file"
+	"github.com/open-telemetry/opentelemetry-log-collection/pipeline"
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/config/configmodels"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/stanzareceiver"
+)
+
+const (
+	typeStr = "filelog"
+	verStr  = "0.14.0"
+)
+
+// NewFactory creates a factory for filelog receiver
+func NewFactory() component.ReceiverFactory {
+	return stanzareceiver.NewFactory(ReceiverType{})
+}
+
+// ReceiverType implments stanzareceiver.LogReceiverType
+// to create a file tailing receiver
+type ReceiverType struct {
+	stanzareceiver.LogReceiverType
```

Comment:
Why are we embedding `stanzareceiver.LogReceiverType` here? I don't see that we use the forwarding of methods that would warrant embedding. I think this line is superfluous (I may be wrong).

Reply:
Good catch. I think this was an artifact of an early effort to make it compile. Will remove.